lib/ProgramOptions/ProgramOptions.h

```diff
@@ -267,6 +267,8 @@ class ProgramOptions {
   std::string _more;
   // context string that's shown when errors are printed
   std::string _context;
+  // already seen to flush programm options
+  std::vector<std::string> _seenParams;
```

Comment:
Should we rename this to `_alreadyFlushed` and change it to a set/unordered_set?